clar2wasm/tests/wasm-generation/main.rs

```diff
@@ -381,6 +382,63 @@ fn qualified_principal() -> impl Strategy<Value = Value> {
     })
 }
 
+fn value_to_string(value: &PropValue, to_response: bool) -> String {
```

Comment:
Imagine being someone who didn't write this function. Why would we have this function when we know that a `PropValue` implements `Display`? Why converting a `Value` to a `String` would care if it's a response or not?

Reply:
This is to ensure that last value is always "stringified" response when function is public. Public functions require last value to be response

Reply:
Where in the name do you understand that the stringified function is a response for a public function?